<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DX11RenderEngineClass</class>
 <widget class="QMainWindow" name="DX11RenderEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DX11RenderEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="RenderViewport" name="renderView" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>371</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabSpacing">
     <number>7</number>
    </property>
    <widget class="QWidget" name="page_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>470</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Home</string>
     </attribute>
     <widget class="QPushButton" name="bt_ChooseBC">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>ChoseBackColor</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>470</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Camera</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>0</y>
        <width>281</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>RotateX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>RotateY</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>RotateZ</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QScrollBar" name="hSBar_CameraRX">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QScrollBar" name="hSBar_CameraRY">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QScrollBar" name="hSBar_CameraRZ">
         <property name="minimum">
          <number>-180</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>470</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Model</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>361</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dSB_MTX">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>RotateL</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QDoubleSpinBox" name="dSB_MSY">
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="dSB_MTZ">
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>RotateW</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QDoubleSpinBox" name="dSB_MRY">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="dSB_MSX">
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="dSB_MRZ">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Translate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="dSB_MSZ">
         <property name="value">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="dSB_MTY">
         <property name="minimum">
          <double>-10000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>10000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dSB_MRX">
         <property name="minimum">
          <double>-180.000000000000000</double>
         </property>
         <property name="maximum">
          <double>180.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="CB_Coord">
         <property name="currentText">
          <string>World</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <item>
          <property name="text">
           <string>World</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Object</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bt_test">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderViewport</class>
   <extends>QWidget</extends>
   <header>renderviewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="DX11RenderEngine.qrc"/>
 </resources>
 <connections/>
</ui>
